### DISCUSSION TITLE
2 Way Sync

### DISCUSSION THREAD

**Discussion by andymac4182** [markdown-confluence/markdown-confluence#265] [Ideas]
When syncing both directions what are some features and gotchas that people are concerned about?

**Comment by planefun2** [comment]
I work with a lot of video and images and these are often transcluded 2x in Obsidian - 

- embedded into a note from the /attachments folder
- any such notes are  - from a heading -  then embedded into another note

Being able to sync all this without having the transclusions break would be amazing

**Reply by andymac4182** [reply]
When the transclusions are embedded are you happy to just copy the content in the new page as if it was written in that page instead of using the embed node in Confluence?

**Comment by johnbillings-metservice** [comment]
I've been thinking about 2-Way sync as well.  How it would work.  Use case is, as a DevOps engineer, I want to document an application that is being developed and supported by an operational team.  They would like to have docs in Confluence.  I would like to write the docs in Markdown in the same repo as the application code and publish them to Confluence.  If an Ops team member would like to add something, update or fix a bug in the docs, they should be able to do that using the Confluence interface.  Possibly, the edit in Confluence pushes back a PR to the repo.  Which we can either accept or amend or refuse and push back our update to Confluence.
